src/Angle.cpp

```diff
@@ -1,15 +1,10 @@
-//
-// Created by thijs on 28-2-19.
-//
-
 #include "../include/roboteam_utils/Angle.h"
 #include "../include/roboteam_utils/Vector2.h"
 #include "../include/roboteam_utils/Definitions.h"
-namespace rtt {
 
-    Angle::Angle(double angle)
-        : angle(angle) {
-        this->constrain();
+namespace rtt {
+    Angle::Angle(double angle) : angle(angle) {
```

Comment:
```suggestion
    Angle::Angle(double &angle) : angle(angle) {
```

Reply:
Why a reference to a double value?

Reply:
to prevent a copy

Reply:
The copy constructor was not used...

Reply:
You want to copy it, since the angle gets normalized